dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/enrollments/EnrollmentsTests.java

```diff
@@ -98,36 +98,6 @@ public void beforeEach() {
     loginActions.loginAsAdmin();
   }
 
-  @ParameterizedTest
```

Comment:
Why are these being removed?
They are not in deprecated tracker

Reply:
They were failing so I assumed they were targeting old tracker. The e2e tests are not neatly separated and confusingly named IMHO when it comes to old and new. I got them back. Lets see if they are failing and if so why.